### ISSUE TITLE
Prediction is not good over large data

### ISSUE THREAD

**Issue by DhineshKS04** [WeiNyn/DIETClassifier-pytorch#3]
Hi, When I add more examples the prediction is not good. It was too bad prediction. Did you have any idea about it?

**Comment by WeiNyn** [comment]
I don't know if the problem comes from data or model, make sure your data is clear, you can try on a normal classification model using BERT. If the classification task is good on your data, so the problem comes from the model.
You can try to turn some hyperparameters related to training/validating, early stopping (you can also check the loss distribution for classification and entities recognition in source code).
The model is using Bert base backbone, so you can try another backbone from hugging face models (this method required your understanding in hugging face and model implementation in PyTorch)
